### ISSUE TITLE
Accessibility issue: Scrollable region must have keyboard access on /notebook-ci-testing/notebooks/DrizzlePac/mask_satellite/mask_satellite.html

### ISSUE THREAD

**Issue by dulude** [spacetelescope/notebook-ci-testing#281]
## What
An accessibility scan flagged 16 elements on https://spacetelescope.github.io/notebook-ci-testing/notebooks/DrizzlePac/mask_satellite/mask_satellite.html because scrollable region must have keyboard access. Learn more about why this was flagged by visiting https://dequeuniversity.com/rules/axe/4.11/scrollable-region-focusable?application=playwright.

The following elements need attention:

- `<pre id="codecell0">` (selector: `#codecell0`)
- `<pre id="codecell3">` (selector: `#codecell3`)
- `<pre id="codecell9">` (selector: `#codecell9`)
- `<pre id="codecell14"><span></span><span class="n">flc_file</span> <span class="o">=</span> <span class="s1">&apos;jc8mb7tcq_flc.fits&apos;</span>        <span class="c1"># defining this variable since we&apos;ll use it a lot</span>
</pre>` (selector: `#codecell14`)
- `<pre id="codecell18">` (selector: `#codecell18`)
- `<pre id="codecell19">` (selector: `#codecell19`)
- `<pre id="codecell25">` (selector: `#codecell25`)
- `<pre id="codecell32"><span></span>CRDS - INFO -  Symbolic context &apos;hst-latest&apos; resolves to &apos;hst_1285.pmap&apos;
CRDS - INFO -  Reorganizing 0 references from &apos;instrument&apos; to &apos;flat&apos;
CRDS - INFO -  Reorganizing from &apos;instrument&apos; to &apos;flat&apos; cache,  removing instrument directories.
</pre>` (selector: `#codecell32`)
- `<pre id="codecell38">` (selector: `#codecell38`)
- `<pre id="codecell41">` (selector: `#codecell41`)
- `<pre id="codecell43">` (selector: `#codecell43`)
- `<pre id="codecell49"><span></span><span class="n">flc_file</span> <span class="o">=</span> <span class="s1">&apos;id8m11g4q_flc.fits&apos;</span>  <span class="c1"># defining this variable since we&apos;ll use it a lot</span>
</pre>` (selector: `#codecell49`)
- `<pre id="codecell53">` (selector: `#codecell53`)
- `<pre id="codecell54">` (selector: `#codecell54`)
- `<pre id="codecell68">` (selector: `#codecell68`)
- `<pre id="codecell72">` (selector: `#codecell72`)

To fix this, ensure elements that have scrollable content are accessible by keyboard in safari.

Specifically:

Fix any of the following:
-   Element should have focusable content
-   Element should be focusable

## Acceptance Criteria
- [ ] The specific violation reported in this issue is no longer reproducible.
- [ ] The fix MUST meet WCAG 2.2 guidelines OR the accessibility standards specified by the repository or organization.
- [ ] A test SHOULD be added to ensure this specific violation does not regress.
- [ ] This PR MUST NOT introduce any new accessibility issues or regressions.
